packages/api/src/routers/GraphQL/resolvers/User/UserAuthInput.ts

```diff
@@ -0,0 +1,12 @@
+import { InputType, Field } from "type-graphql";
+
+@InputType()
+class UserAuthInput {
+  @Field()
+  emailAddress: string;
+
+  @Field()
+  password: string;
+}
```

Comment:
Keep things well structured and strongly typed by using input for all mutations.